sdk/account_test.go

```diff
@@ -60,82 +63,65 @@ const accountInfoJson = `
    }
 }
 `
+)
 
-func TestAccountService_GetAccountInfo(t *testing.T) {
-	cl, mux, _, teardown, err := setupMockServer()
-	if err != nil {
-		t.Errorf("Account.GetAccountInfo error setting up mock server: %v", err)
-	}
-	defer teardown()
+var (
+	testAddress1 = "SAONSOGFZZHNEIBRYXHDTDTBR2YSAXKTITRFHG2Y"
```

Comment:
It would be nice for us to have meaningful names like HEM-address in order to understand or replace the values in tests in the future.